packages/@ember/object/type-tests/evented/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { expectTypeOf } from 'expect-type';
+
+import Mixin from '@ember/object/mixin';
+import Evented from '@ember/object/evented';
+
+// A very naive test that at least makes sure we can import this
```

Comment:
This is good; any chance we can also pull over some of the type tests [from DT](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ember__object/test/event.ts) here? Me, I want Evented burned to the ground, but if we’re shipping types for it here, we should cover the ground. 😩

Reply:
I thought I had, but we can bring over more if I missed some.